microsoft/electionguard-python
# #695 Create Decryption

### Issue

Fixes #695 

### Description
Adds the ability to create decryptions for an election and view those decryptions.  It does not yet include the ability for guardians to join a decryption.

### Testing
1. Create an election
2. Click the "Create Decryption" button at the top right

Expected: It should navigate to a create decryption page and the name should be auto-filled in with a suggested value

![image](https://user-images.githubusercontent.com/1769905/181105406-f658cf28-b464-4460-9561-93be20a147f7.png)

3. Click create

Expected: It should navigate to a view decryption page like this:

![image](https://user-images.githubusercontent.com/1769905/181105330-1af8b87d-59bf-4fd3-b2df-611458e451c6.png)
